nixos/modules/services/mail/dovecot.nix

```diff
@@ -25,6 +25,7 @@ let
       ssl_cert = <${cfg.sslServerCert}
       ssl_key = <${cfg.sslServerKey}
       ${optionalString (!(isNull cfg.sslCACert)) ("ssl_ca = <" + cfg.sslCACert)}
+      ssl_dh = </var/lib/dhparams/dovecot2.pem
```

Comment:
Should use `${config.security.dhparams.path}` instead of `/var/lib/dhparams` here, in order to be resilient to change ;)